### PR TITLE
Update the house routes with queries for location

### PR DESCRIPTION
**Pull request by Asayaobi**

### CODE REVIEW

**Review by ynotrusso — DISMISSED** [review]
Please test your routes before pushing. Now you have a query that says `SELECT * FROM houses WHERE 1 = 1 WHERE location = ...` with a double `WHERE`.

File: routes/housesRoutes.js [hunk 1]
```diff
@@ -23,10 +23,17 @@ router.get('/houses', async (req, res) => {
...
     let queryString = 'SELECT * FROM houses'
+    if (!req.query.location) {
+      queryString
+    }
     //query for location
     if (req.query.location) {
       queryString += ` WHERE location = '${req.query.location}'`
     }
+    //query for max price
+    if (req.query.max_price) {
```
[the remark this answers is not in the document]
Comment (Asayaobi) [reply routes/housesRoutes.js:31]: Thank you for the hints, Tony. After 2 cans of Leo, I think I managed to get it working.. hopefully

**Review by enaros — CHANGES_REQUESTED** [review]
look at the order, it is hardcoded

**Review by enaros — APPROVED** [review]
LGTG!

**Pull request merged**